CovertInfiltration/Config/XComEncounterLists.ini

```diff
@@ -0,0 +1,9 @@
+[XComGame.XComTacticalMissionManager]
++SpawnDistributionLists=(ListID="PhalanxLeaders", \\
+   SpawnDistribution[0]=(Template="AdvShieldBearerM2", MinForceLevel=8,  MaxForceLevel=13, MaxCharactersPerGroup=1, SpawnWeight=31), \\
+   SpawnDistribution[1]=(Template="AdvShieldBearerM3", MinForceLevel=14, MaxForceLevel=20, MaxCharactersPerGroup=1, SpawnWeight=56))
+
++SpawnDistributionLists=(ListID="CongregationLeaders", \\
+	SpawnDistribution[0]=(Template="AdvPriestM1",       MinForceLevel=3,  MaxForceLevel=7,  MaxCharactersPerGroup=1, SpawnWeight=15), \\
```

Comment:
Since `PhalanxLeaders` has nothing until FL8 and `CongregationLeaders` has nothing till FL3, set min FL on those sitreps